torch/testing/_internal/common_utils.py

```diff
@@ -112,13 +112,19 @@
 disabled_tests_dict = {}
 slow_tests_dict = {}
 
+def maybe_load_json(filename):
+    if os.path.exists(filename):
```

Comment:
Not sure if we want to raise an exception here if somebody to call this function with "." or ".."
```suggestion
    if os.path.isfile(filename):
```